docker/start.sh

```diff
@@ -143,10 +143,10 @@ register_default_namespace() {
     echo "Temporal CLI Address: $TEMPORAL_CLI_ADDRESS"
     sleep 5
     echo "Registering default namespace: $DEFAULT_NAMESPACE"
-    until tctl --do $DEFAULT_NAMESPACE namespace describe < /dev/null; do
+    until tctl --ns $DEFAULT_NAMESPACE namespace describe < /dev/null; do
```

Comment:
I checked that env variable name was updated but forgot to update flags. My bad.